tests/patches/glpi-allow-test-on-master-branch.patch

```diff
@@ -1,13 +1,12 @@
 diff --git a/setup.php b/setup.php
-index eed6660..1f065eb 100644
+index eed6660..79c56de 100644
 --- a/setup.php
 +++ b/setup.php
-@@ -209,7 +209,7 @@ function plugin_version_flyvemdm() {
-       'requirements'   => [
-          'glpi' => [
-             'min' => $glpiVersion,
--            'max' => PLUGIN_FLYVEMDM_GLPI_MAX_VERSION,
-+            'max' => '9.5',
```

Comment:
removing this hunk should be sufficient if you change the value of ```PLUGIN_FLYVEMDM_GLPI_MAX_VERSION```, am I right ?

Reply:
Yes, I've already test it with that change on another local installation.